src/EnergyPlus/WindowManager.cc

```diff
@@ -7371,7 +7379,7 @@ namespace WindowManager {
                         state.dataConstruction->Construct(ThisNum).VisTransNorm = TransVisNorm;
                         state.dataConstruction->Construct(ThisNum).SolTransNorm = TransSolNorm;
 
-                        static constexpr std::string_view Format_700(" WindowConstruction,{},{},{},{},{:.3R},{:.3R},{:.3R},{:.3R}\n");
+                        static constexpr std::string_view Format_700(" WindowConstruction,{},{},{},{},{:.3R},{:.3R},{:.3R},{:.3R},{:.3R},{:.3R}\n");
```

Comment:
Extend format so that Solar and Visible Transmittance values are written to eio for the WindowConstruction report.